src/test/java/javaslang/collection/euler/ProjectEulerTest.java

```diff
@@ -85,9 +84,37 @@ private static long fibonacci(int order) {
      * <p>See also <a href="https://projecteuler.net/problem=7">projecteuler.net problem 7</a>.</p>
      */
     @Test
-    @Ignore
-    public void shouldSolveProblem7() {
-        final int actual = Primes.asStream().drop(10_000).head();
-        assertThat(actual).isEqualTo(0); // TODO, see Primes#sieve(Stream)
+    public void test7() {
```

Comment:
We should choose similar names for the test methods. I like the `shouldXxx` notion because is a readable spec and `testXxx` is redundant with `@Test`.